Makefile

```diff
@@ -156,6 +156,13 @@ e2e-upgrade: $(GINKGO) $(KUSTOMIZE) $(KIND) $(GOVC) ## Run e2e tests
 	@echo
 
 	time $(GINKGO) -v -focus="ClusterAPI Upgrade Tests" ./test/e2e -- --e2e.config="$(E2E_CONF_FILE)"
+
```

Comment:
A good thing to do to avoid replication is move the TEST_ARGS variable to the `make test` target and as part of `make test-cover` call `make test` with the TEST_ARGS set to run test coverage data. Then execute the 2 tasks to at lines 164 and 165

Reply:
@geetikabatra Can you also take a look at this comment?

Reply:
The TESt_ARGS needs to be set before running test-cover otherwise no coverage files will get generated